test/integration/members.test.js

```diff
@@ -22,6 +22,10 @@ const nonSuperUser = userData[0];
 const userDoesNotExists = userData[1];
 const userToBeArchived = userData[3];
 const userAlreadyArchived = userData[5];
+const userWithRolesObjectWithoutArchivedProperty = userData[0];
+const userWithoutRolesObject = userData[1];
+const userWithRolesObjectWithArchivedTrue = userData[5];
+const userWithRolesObjectWithArchivedFalse = userData[6];
```

Comment:
short variable name from the long variable name.

Reply:
Sure. We can shorten them a bit. Done that in the later PR https://github.com/Real-Dev-Squad/website-backend/pull/654